test/componentarrays.jl

```diff
@@ -93,6 +93,40 @@ soln(0.9)
 #     ob.color = findnode(soln(t), :C)[point_map]
 # end
 
+function closest_point(p1, p2, dims)
```

Comment:
Delete

Reply:
Can't this is used in flat_op

Reply:
Removing these should become an issue and a separate PR. I would consider removing this not in the scope of this PR.